test/collectionUtils/AnyResourceTransformer.test.ts

```diff
@@ -90,4 +92,13 @@ describe("AnyResourceTypeTransformer", () => {
       "/tmp/FHIR/AnyResource"
     );
   });
+
+  it("should keep the IResource import if the interface extends IResource", () => {
+    builder.parentType = iResourceType;
+    const newEntity = builder.buildEntityDefinition();
+    const result = transformer.transform(newEntity);
+    expect(result.imports.dataTypes).toBeArrayOfSize(2);
+    expect(result.imports.dataTypes[0].normalizedName).toBe("AnyResource");
+    expect(result.imports.dataTypes[1].normalizedName).toBe("IResource");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5970468)